dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -229,6 +230,8 @@ export const Liveness = ({
 		if (!topOfBlog) return () => {};
 
 		const observer = new window.IntersectionObserver(([entry]) => {
+			if (!entry) return;
+
```

Comment:
Can you have an empty entry? This might have to be improved in the typescript lib itself.